eval/mt_bench/components.py

```diff
@@ -184,16 +187,17 @@ def stop_vllm():
     outputs = NamedTuple("outputs", best_model=str, best_score=float)
     best_model = max(scores, key=scores.get)
     best_score = scores[best_model]
-    if best_score_file:
-        with open(best_score_file, "w", encoding="utf-8") as f:
-            json.dump({"best_model": best_model, "best_score": best_score}, f, indent=4)
-
     # Rename the best model directory to "candidate_model" for the next step
     # So we know which model to use for the final evaluation
     os.rename(
         os.path.join(models_path_prefix, best_model),
         os.path.join(models_path_prefix, "candidate_model"),
     )
+    best_model = f"{models_path_prefix}/candidate_model"
```

Comment:
please use `os.path.join` instead of string formatting.

Reply:
Also, why don't we print the "sample_" instead? If we print "candidate_model", the user has no idea which model was the best I guess...

Reply:
I thought about this - but we rename it to `candidate_model` so `samples_` no longer exists, right? What's saved to S3 is "candidate_model" right?